### PR TITLE
Use public api to get open position info

### PR DESCRIPTION
**Pull request by ukorvl**
This pull request is based on #146 
This diff eliminates usage of freshteam api token, because it is not secure, event with recruiter privileges.
There is open api which freshteam widget utilizes under the hood. This diff makes site compatible with that api.

### CODE REVIEW

File: site/src/pages/careers/jobs/[id].tsx [hunk 1]
```diff
@@ -21,23 +22,29 @@ export async function getStaticProps({ params }: GetStaticPropsContext<{ id: str
...
   if (!jobPosting) {
     return {
       notFound: true,
     }
   }
 
+  const { job_roles, branches } = jobInfo
+  const mappedJob = mapRawJobToUIJob(jobPosting, job_roles, branches, false)
+
+  console.log(jobPosting)
```
Comment (olegrok) [inline site/src/pages/careers/jobs/[id].tsx:39]: Debug?

File: site/next.config.js [hunk 2]
```diff
@@ -9,6 +9,22 @@ const nextConfig = {
...
+          },
+        ],
+        permanent: false,
+      }
+    ];
+  }
 };
 
-module.exports = nextConfig;
+module.exports = nextConfig;
```
Comment (olegrok) [inline site/next.config.js:30]: It's better to return empty newline back

**Review by olegrok — APPROVED** [review]

File: site/src/pages/careers/jobs/[id].tsx (hunk 1 above)
Comment (ukorvl) [reply site/src/pages/careers/jobs/[id].tsx]: Removed

File: site/next.config.js (hunk 2 above)
Comment (ukorvl) [reply site/next.config.js]: Done

**Pull request merged**